pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -369,7 +369,7 @@ func (s *Server) createIstioCA(client corev1.CoreV1Interface, opts *CAOptions) (
 		s.caBundlePath = certChainFile
 
 		caOpts, err = ca.NewPluggedCertIstioCAOptions(certChainFile, signingCertFile, signingKeyFile,
```

Comment:
Every time I see the long list of parameters I want to do something... Worse - there are 4 places where this gets copied. 

For agent I started to create a struct and pass ProxyConfig - most of those things should actually be part of ProxyConfig/MeshConfig anyways.